dev-python/pytest-asyncio/pytest-asyncio-0.9.0.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python3_{5,6,7} )
+
+inherit distutils-r1
+
+DESCRIPTION="Pytest support for asyncio"
+HOMEPAGE="https://github.com/pytest-dev/pytest-asyncio https://pypi.org/project/pytest-asyncio/"
+SRC_URI="https://github.com/pytest-dev/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="Apache-2.0"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-python/async_generator[${PYTHON_USEDEP}]
+"
+
+DEPEND="
```

Comment:
You also need RDEPEND during tests otherwise they fail because of this.
```py
    extras_require={
        ':python_version == "3.5"': "async_generator >= 1.3",
```